include/common_package/node_names.hpp

```diff
@@ -0,0 +1,34 @@
+//
+// Created by Johan <job8197@thi.de> on 17.05.2024.
+//
+
+#ifndef THI_DRONE_WS_NODE_NAMES_HPP
+#define THI_DRONE_WS_NODE_NAMES_HPP
+
+#include <set>
+
+namespace common_lib::node_names {
+
+constexpr char const *const MISSION_CONTROL =
+    "mission_control"; /**< Node name for the mission control node */
+
+constexpr char const *const WAYPOINT =
+    "waypoint"; /**< Node name for the navigate to waypoint node */
+
+constexpr char const *const QRCODE_DETECTION =
+    "qrcode_detection"; /**< Node name for the qr code detection node */
+
+constexpr char const *const FCC_BRIDGE =
+    "fcc_bridge"; /**< Node name for the FCC bridge node */
+
+constexpr char const *const TELEMETRY =
```

Comment:
I don't think we need this one here, as it is not relevant to the rest of the nodes.

Reply:
I can leave it out. I just thought the telemetry node is using common lib anyway for the topic names and then we can define its name here as well.

Reply:
Yes, it is using the common lib but I think we should leave the telemetry node out of our production code

Reply:
Fixed in 5ca296d